blobstore/src/test/java/org/jclouds/blobstore/strategy/internal/ListContainerTest.java

```diff
@@ -72,7 +72,7 @@ public void testListWithPrefix() {
       blobStore.putBlob(containerName, blobStore.blobBuilder("bar").payload("").build());
 
       Iterable<? extends StorageMetadata> results = concatter.execute(containerName,
-            ListContainerOptions.Builder.prefix(prefix).recursive());
+            ListContainerOptions.Builder.prefix(prefix)./* irrelevant */recursive());
```

Comment:
Stray comment?

Reply:
No, noting that this test passes regardless of which way you set the `recursive` flag, which makes the name of the test a bit surprising.